src/content/glossary/index.md

```diff
@@ -473,6 +473,12 @@ A [rollup](#rollups) of transactions that use [fraud proofs](#fraud-proof) to of
 
 <DocLink to="/developers/docs/scaling/layer-2-rollups/#optimistic-rollups" title="Optimistic Rollups" />
 
+### Oracle {#oracle}
+
+An oracle is a bridge between the [blockchain](#blockchain) and the real world. They act as on-chain [APIs](#api) that can be queried for information and used in [smart contracts](#smart-contract). This could be anything from price information to weather reports. Oracles can also be bi-directional, used to "send" data out to the real world.
```

Comment:
Agreed @wackerow I also think we could drop the example here as we link out to the same Oracle's doc which has the same example.

```suggestion
An oracle is a bridge between the [blockchain](#blockchain) and the real world. They act as on-chain [APIs](#api) that can be queried for information and used in [smart contracts](#smart-contract).
```